apps/charterafrica/.env.template

```
@@ -4,6 +4,7 @@ MONGO_URL
 
 PAYLOAD_PUBLIC_APP_URL
 PAYLOAD_SECRET_KEY
+PAYLOAD_ALLOWED_DOMAINS=
```

```suggestion
PAYLOAD_CORS=
```
1. This is not for domains but full URLs.
2. If it needs to be accessible to the front-end (similar to `APP_URL`), it must have PUBLIC prefix e.g. `PAYLOAD_PUBLIC_CORS`.
3. Order env vars alphabetically.